shift-circuit-breaker.gemspec

```diff
@@ -27,10 +27,10 @@ Gem::Specification.new do |s|
   s.test_files    = `git ls-files -- { test, spec, features }/*`.split("\n")
   s.executables   = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
 
-  s.add_dependency "activesupport", ">= 3.0.0"
-  s.add_dependency "bundler", ">= 1.16.1"
-  s.add_dependency "newrelic_rpm", ">= 3.1"
-  s.add_dependency "sentry-raven", ">= 1.2.3"
+  s.add_runtime_dependency "activesupport", "~> 5.1", ">= 5.1.4"
+  s.add_runtime_dependency "bundler", "~> 1.16"
```

Comment:
Isn't bundler just needed for dev?

Reply:
cheers @ryantownsend, that's right - will update 👍